storage_service/locations/tests/test_package.py

```diff
@@ -378,6 +379,35 @@ def test_run_post_store_callbacks_dip(self):
             body = '{"download_url": "http://ss.com/api/v2/file/%s/download/"}' % uuid
             mocked_execute.assert_called_with(url, body)
 
+    @staticmethod
```

Comment:
I'm wondering what are the implications in using staticmethod in TestCase. One issue is that you won't have access to its [methods](https://docs.python.org/3/library/unittest.html#unittest.TestCase) but that's obviously not a problem since we use pytest most of the time. But I'm wondering if there are unknowns like problems around transaction management, probably not but I figured I'd raise it just in case you know more.

Reply:
I think interesting to meditate on, but I don't think we'll see an impact here.